packages/SwingSet/test/zcf-ish-upgrade/test-zcf-ish-upgrade.js

```diff
@@ -9,9 +9,10 @@ import { initializeSwingset, makeSwingsetController } from '../../src/index.js';
 const bfile = name => new URL(name, import.meta.url).pathname;
 
 test('zcf-ish upgrade', async t => {
+  /** @type {SwingSetConfig} */
   const config = {
     includeDevDependencies: true, // for vat-data
-    defaultManagerType: 'local', // 'xs-worker',
+    defaultManagerType: 'local',
```

Comment:
I'm not confident in speaking for everybody that we no longer want this handy.

In particular, git shows that this came from e9240372c7 by @warner , reviewed by @Chris-Hibbert , so you folks are invited to weigh in.

Reply:
likewise https://github.com/Agoric/agoric-sdk/pull/7103/commits/9a1ed1ed08ba59f941f81a27bfb2362794f15d4b#diff-39b592732cda179faf8061030bd0a81f099804ed9c1db6d4dddc85bc958272abL14

Reply:
With the `SwingsetConfig` type annotation alternative values will now autocomplete. The comment was just there to tell you the alternate values so now it's redundant noise imo.